### PR TITLE
Verify irqbalance tuning is reverted

### PR DESCRIPTION
**Pull request by gsr-shanks**
https://bugzilla.redhat.com/show_bug.cgi?id=1854245

### CODE REVIEW

File: functests/2_performance_update/updating_profile.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 	chkKernel := []string{"uname", "-a"}
 	chkCmdLine := []string{"cat", "/proc/cmdline"}
 	chkKubeletConfig := []string{"cat", "/rootfs/etc/kubernetes/kubelet.conf"}
+	chkIrqbalance := []string{"grep", "-ls" + "ffffffff,fffffffe" + "/etc/sysconfig/irqbalance"}
```
Comment (fromanirh) [inline functests/2_performance_update/updating_profile.go:33]: not sure the "+" is correct here. Shouldn't at least the last component (`/etc/sysconfig/irqbalance`) separated by a comma?

Comment (gsr-shanks) [reply functests/2_performance_update/updating_profile.go]: Fixed. Thanks.

File: functests/2_performance_update/updating_profile.go [hunk 2]
```diff
@@ -269,6 +270,10 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			kblcfg, err := nodes.ExecCommandOnNode(chkKubeletConfig, newCnfNode)
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to execute %s", chkKubeletConfig))
 			Expect(kblcfg).NotTo(ContainSubstring("reservedSystemCPUs"))
+
+			irqbal, err := nodes.ExecCommandOnNode(chkIrqbalance, newCnfNode)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to execute %s", chkIrqbalance))
+			Expect(irqbal).NotTo(ContainSubstring("ffffffff,fffffffe"))
```
Comment (dshchedr) [inline functests/2_performance_update/updating_profile.go:276]: "ffffffff,fffffffe" - is only for scenario with  profile parameter  **reserved: "0"**
So, this test may not work correctly on cluster with different "reserved" value. 

How about adding calculation of this value based on profile setting?

**Comment by marcel-apf** [comment]
> BZ1854245

II would use a real link for the BZ, especially for the lazy people around us :)

**Comment by gsr-shanks** [comment]
> > BZ1854245
> 
> II would use a real link for the BZ, especially for the lazy people around us :)

Sure I am one of them :) 
I was hoping(lazy) that it would somehow auto-link itself. Will fix it.

File: functests/2_performance_update/updating_profile.go (hunk 2 above)
Comment (gsr-shanks) [reply functests/2_performance_update/updating_profile.go:276]: Agree it is good to have it this way.

but before that, it needs to be investigated why we see the following in ci/prow/e2e-gcp which probably means that on reverting the label the file is being removed in which case we would need to check if the file exists.

```
  out=cat: /etc/sysconfig/irqbalance: No such file or directory
```

Comment (gsr-shanks) [reply functests/2_performance_update/updating_profile.go:276]: Missed to add _/rootfs_ prefix for the no such file error. Fixed it.

**Pull request merged**